R-proj/test_ess.R

```diff
@@ -0,0 +1,5 @@
+library(volesti)
```

Comment:
That file should not be in the PR right?

Reply:
Thank you for your feedback and comments. Yes I removed this file.